### ISSUE TITLE
Dependabot is suggesting PRs to update L7.1.x to L7.30.x that break

### ISSUE THREAD

**Issue by assertchris** [facade/ignition#363]
I don't know at which point Laravel 7 changed the container bind signature; but I just merged a Dependabot PR that broke my site. I should have checked before merging (doh!) but it seems that the version upgrade Dependabot suggests introduces a breaking change to the container bind syntax; and the only package that caused an issue is the ignition facade.

**To be clear: I don't expect any change to be made because this is for an older version of Laravel. Just raising it because Freek suggested I do, and in case someone else comes across the same situation.**

The solution, for me, was just to remove the ignition/facade package; because I don't use it on this site. 

Before Dependabot (working): https://github.com/assertchris/assertchris.io/blob/a413335ad3d581b57039227fb33be476484c45a1/composer.json
After Dependabot (not working): https://github.com/assertchris/assertchris.io/blob/55ef9f8f90cb3c45aa4725e01b97b172d2bb94aa/composer.json

**Comment by AlexVanderbist** [comment]
Hi Chris! Thanks for bringing this to our attention & sorry you ran into this. I've looked into this and it seems to have been fixed in Ignition v2.3.2 and above. As Dependabot only upgrades the composer.lock file for the affected dependency this update was missing in your app. It's not ideal for these Dependabot issues on older Laravel versions but `composer update` probably would've fixed it.

tl;dr for anyone else with this problem: `composer update` and make sure you're using an Ignition version >= `2.3.2`.
